Source/World/Chunk/ChunkSection.cpp

```diff
@@ -81,6 +81,7 @@ void ChunkSection::bufferMesh()
 {
     m_meshes.solidMesh.bufferMesh();
     m_meshes.waterMesh.bufferMesh();
+    m_meshes.waterMesh.bufferMesh();
```

Comment:
This is repeated twice?

Reply:
I fixed this before merging into master :)